modules/nw-dpu-installing-operator.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/installing-dpu-operator.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-dpu-operator_{context}"]
+= Installing the DPU Operator
+
+The DPU Operator on the host cluster side is responsible for the life-cycle management of the `ovn-kube` components and the necessary host network initialization on IPU.
+
+As a cluster administrator, you can install the DPU Operator by using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== CLI: Installing the DPU Operator
+
+As a cluster administrator, you can install the DPU Operator by using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* An account with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `openshift-dpu-operator` namespace by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF| oc create -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-dpu-operator
+  annotations:
+    workload.openshift.io/allowed: management
+EOF
+----
+
+. Create an `OperatorGroup` custom resource (CR) by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF| oc create -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: dpu-operators
+  namespace: openshift-dpu-operator
+spec:
+  targetNamespaces:
+  - openshift-dpu-operator
+EOF
+----
+
+. Create a `Subscription` CR for the SR-IOV Network Operator by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF| oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-dpu-operator-subscription
+  namespace: openshift-dpu-operator
+spec:
+  channel: stable
+  name: dpu-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.Verification
+
+. Check that the Operator is installed by entering the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-dpu-operator \
+  -o custom-columns=Name:.metadata.name,Phase:.status.phase
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+Name                                Phase
+dpu-operator.v4.18.0-202503130333   Succeeded
+----
+
+. Change to the `openshift-dpu-operator` project:
++
+[source,terminal]
+----
+$ oc project openshift-dpu-operator
+----
+
+. Verify the DPU Operator is running by entering the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-dpu-operator
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+NAME                                               READY   STATUS    RESTARTS   AGE
+dpu-operator-controller-manager-6b7bbb5db8-7lvkj   2/2     Running   0          2m9s
+----
+
+[id="install-dpu-operator-web-console_{context}"]
+== Installing the DPU Operator using the web console
+
+As a cluster administrator, you can install the DPU Operator by using the web console.
```

Comment:
when installing on the host side.

For the DPU side, the CLI must be used since MicroShift doesn't come with a web console.